install/kubernetes/helm/istio/charts/jaeger/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
What is this file doing ? We don't seem to have it in the other packages.

Reply:
It was in the zipkin folder, so just copied it - but could remove.